tests/Models/Casting.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MongoDB\Laravel\Tests\Models;
+
+use MongoDB\Laravel\Eloquent\Model as Eloquent;
+
+class Casting extends Eloquent
+{
+    protected $connection = 'mongodb';
+    protected $collection = 'casting';
+
+    protected $fillable = [
+        'intNumber',
+        'floatNumber',
+        'decimalNumber',
+        'stringContent',
+        'stringContent',
+        'booleanValue',
+        'objectValue',
+        'jsonValue',
+        'collectionValue',
+        'dateField',
+        'datetimeField',
+    ];
+
+    protected $casts = [
+        'intNumber' => 'int',
+        'floatNumber' => 'float',
+        'decimalNumber' => 'decimal:2',
```

Comment:
Is there a case with `decimal` without the prevision part?

Reply:
As mentioned in the #2146 issue, an error will be thrown.